### PR TITLE
{bio}[system/system] Gctf v1.06

### PR DESCRIPTION
**Pull request by akesandgren**
(created using `eb --new-pr`)

Requires easyblock PR https://github.com/easybuilders/easybuild-easyblocks/pull/1827

### CODE REVIEW

**Comment by smoors** [comment]
@akesandgren I was working on this as well. I think (no reply from the user yet) I solved it without wrappers by adding the following lines (the binaries seem to depend on libcufft only):
```
# make sure the required libs are present even when another version of CUDA is loaded
postinstallcmds = [
    'cd %(installdir)s && mkdir lib && cd lib && ln -s $EBROOTCUDA/lib64/libcufft.so.8.0',
]
```

**Comment by akesandgren** [comment]
Ohhh, that's ugly :-)

**Comment by smoors** [comment]
yes a dirty hack, I agree :)

File: easybuild/easyconfigs/g/Gctf/Gctf-1.06.eb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+homepage = 'https://www.mrc-lmb.cam.ac.uk/kzhang/'
+description = """Gctf: real-time CTF determination and correction, Kai Zhang, 2016"""
+
+toolchain = SYSTEM
+
+source_urls = ['http://www.mrc-lmb.cam.ac.uk/kzhang/Gctf/']
+sources = [
+    '%(name)s_v%(version)s_and_examples.tar.gz',
+]
+patches = [('Gctf-wrapper.sh', '.')]
```
Comment (boegel) [inline easybuild/easyconfigs/g/Gctf/Gctf-1.06.eb]: @akesandgren This line should be removed now the custom easyblock is used?

Comment (akesandgren) [reply easybuild/easyconfigs/g/Gctf/Gctf-1.06.eb]: Gah, I thought I had done that... fixed.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2604.swalot.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/4556ed7b914d784000ee85486523bbb6 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3112.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/592ffa86874433475c2bec8f6dde6f27 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2635.swalot.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/489a2570bd88cb6524c8424dad92e7b1 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @akesandgren!

**Pull request merged**